src/QsFmt/Formatter/SyntaxTree/Statement.fsi

```diff
@@ -141,75 +175,65 @@ type internal Let =
     }
 
 /// <summary>
-/// A <c>return</c> statement.
+/// An <c>update</c> statement.
 /// </summary>
-type internal Return =
+type internal UpdateStatement =
```

Comment:
Is this used for both 'set a = 5;' and for 'set a += 5;'? 
If it is, then the structure of this statement doesn't cover the 'set (a,b) = (1,2);'. If it is not, then where is the SetStatement? 
Edit: Looks like it isn't, and I've found the SetStatement below.